lib/looker.ts

```diff
@@ -47,6 +47,7 @@ export async function runLookQuery(lookId: string): Promise<string> {
 export async function runQueryForTemplate(
   template: string,
   filters: any,
+  filter_expression?: any,
```

Comment:
Please add JSDoc specifying what this function does.  Be sure to describe how filters and filter_expression interact.